manager/manager.go

```diff
@@ -148,11 +148,15 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, houskeepingConfig
 	}
 
 	// Detect the container we are running on.
-	selfContainer, err := cgroups.GetOwnCgroupPath("cpu")
-	if err != nil {
-		return nil, err
+	selfContainer := "/"
+	var err error
+	if !cgroups.IsCgroup2UnifiedMode() {
```

Comment:
why doesn't this work in cgroup2 unified mode?

Reply:
GetOwnCgroup just fails on cgroup v2 at the moment:

https://github.com/opencontainers/runc/blob/dd8d48ede80d6538cab42673699a4f6704e58912/libcontainer/cgroups/utils.go#L312-L314

Reply:
Can we add a warning if running in unified mode, and open an issue to track removing this once getowncgroup is working?

Reply:
added a warning and reported the issue to opencontainers/runc